.circleci/config.yml

```diff
@@ -328,6 +337,33 @@ jobs:
           name: Code coverage
           command: bash <(curl -s https://codecov.io/bash)
 
+  docker-otelcol:
+    docker:
+      - image: cimg/go:1.15
+    steps:
+      - attach_to_workspace
+      - setup_remote_docker
+      - run:
+          name: Build docker image
+          command: make docker-otelcol SKIP_COMPILE=true
+      - run:
+          name: Run docker image
+          command: |
+            docker run -d -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 --name otelcol otelcol:latest
```

Comment:
Don't need to address now, but it's preferable to not spam us0 and instead use a hosts entry and mock service.

Reply:
Makes sense. I'll just change it to some fake realm for now since we're not validating anything here other than the docker image can start and run.